src/modules/kafka/doc/kafka_admin.xml

```diff
@@ -0,0 +1,221 @@
+		  </itemizedlist>
+		</para>
+		<para>
+		  &lt;proto&gt; is either PLAINTEXT, SSL, SASL, SASL_PLAINTEXT
+		</para>
+		<para>
+		  The two formats can be mixed but ultimately the value of the
+		  <emphasis>security.protocol</emphasis> config property decides what brokers are allowed.
+		</para>
+		<para>
+		  <emphasis>
+			This parameter is mandatory. There is no default value.
+		  </emphasis>
+		</para>
+		<example>
+		  <title>Set <varname>brokers</varname> parameter</title>
+		  <programlisting format="linespecific">
+...
+modparam("kafka", "brokers", "localhost:9092")
+modparam("kafka", "brokers", "broker1:10000,broker2")
+modparam("kafka", "brokers", "SSL://broker3:9000,ssl://broker2")
+...
+		  </programlisting>
+		</example>
+	  </section>
+	  <section id="kafka.p.configuration">
+		<title><varname>configuration</varname> (string)</title>
+		<para>
+		  Specifies a set of general properties.
+		</para>
+		<para>
+		  Each configuration property follows: <emphasis>name = value</emphasis> pattern.
+		  And configuration properties are separated by <emphasis>;</emphasis>
+		</para>
+		<para>
+		  This parameter is optional, but if it exists it can be configured only once.
+		</para>
+		<example>
+		  <title>Set <varname>configuration</varname> parameter</title>
+		  <programlisting format="linespecific">
+...
+modparam("kafka", "configuration", "topic.metadata.refresh.interval.ms=20000;queue.buffering.max.messages=1000000;metadata.request.timeout.ms=90000")
+
+modparam("kafka", "configuration", "topic.metadata.refresh.interval.ms=20000;queue.buffering.max.messages=500000;debug=all;metadata.request.timeout.ms=900000")
+...
+		  </programlisting>
+		</example>
+	  </section>
+	  <section id="kafka.p.topic">
+		<title><varname>topic</varname> (string)</title>
+		<para>
+		  Specifies a topic name and a set of topic properties.
+		</para>
+		<para>
+		  The topic defined in topic parameter has to already exist in Kafka servers.
+		</para>
+		<para>
+		  Each topic property is a list of <emphasis>attribute = value</emphasis> separated by semicolon.
+		</para>
+		<para>
+		  name atribute indicates the topic name. It is mandatory.
+		  Other attributes mean names of properties and are optional.
+		</para>
+		<para>
+		  This parameter is optional. Each topic needs a topic parameter so several topic parameters are allowed.
+		</para>
+		<example>
+		  <title>Set <varname>topic</varname> parameter</title>
+		  <programlisting format="linespecific">
+...
+modparam("kafka", "topic", "name=my_topic;request.required.acks=0;request.timeout.ms=10000")
+modparam("kafka", "topic", "name=second_topic;request.required.acks=0;request.timeout.ms=10000")
+modparam("kafka", "topic", "name=third_topic")
+...
+		  </programlisting>
+		</example>
+	  </section>
+	</section>
+	<section>
+	  <title>Functions</title>
+	  <section id="kafka.f.kafka_send">
+		<title>
+		  <function moreinfo="none">kafka_send(topic, msg)</function>
```

Comment:
Would be good to add a sentence about the return code of the function, that it returns -1 if for all failures etc..